### PR TITLE
Update Parse interceptor comment

### PR DESCRIPTION
**Pull request by wangmengyan95**

### CODE REVIEW

File: Parse/src/main/java/com/parse/Parse.java [hunk 1]
```diff
@@ -608,7 +608,14 @@ private static void initializeParseHttpClientsWithParseNetworkInterceptors() {
...
   }
 
-  // TODO(mengyan) Make public after we release the interceptor feature
+
+  /**
+   * Add a {@code ParseNetworkInterceptor}. You must invoke
+   * {@code Parse.addParseNetworkInterceptor} before {@code Parse.initialize}. You can add multiple
+   * {@code ParseNetworkInterceptor}.
+   * @param interceptor
+   *          The {@code ParseNetworkInterceptor} should be added.
```
Comment (hallucinogen) [inline Parse/src/main/java/com/parse/Parse.java]: `s/should be/to be`

File: Parse/src/main/java/com/parse/Parse.java [hunk 2]
```diff
@@ -620,7 +627,12 @@ private static void initializeParseHttpClientsWithParseNetworkInterceptors() {
     interceptors.add(interceptor);
   }
 
-  // TODO(mengyan) Make public after we release the interceptor feature
+  /**
+   * Remove a given {@code ParseNetworkInterceptor}. You must invoke
+   * {@code Parse.removeParseNetworkInterceptor} before {@code Parse.initialize}.
+   * @param interceptor
+   *          The {@code ParseNetworkInterceptor} should be removed.
```
Comment (hallucinogen) [inline Parse/src/main/java/com/parse/Parse.java]: `s/should be/to be`

File: Parse/src/main/java/com/parse/Parse.java [hunk 3]
```diff
@@ -620,7 +627,12 @@ private static void initializeParseHttpClientsWithParseNetworkInterceptors() {
     interceptors.add(interceptor);
   }
 
-  // TODO(mengyan) Make public after we release the interceptor feature
+  /**
+   * Remove a given {@code ParseNetworkInterceptor}. You must invoke
+   * {@code Parse.removeParseNetworkInterceptor} before {@code Parse.initialize}.
```
Comment (hallucinogen) [inline Parse/src/main/java/com/parse/Parse.java]: ideally this should be `{@link Parse#removeParseNetworkInterceptor()}` though I'm not super familiar with the syntax.

Also it should be `{@link Parse#initialize()}.`

File: Parse/src/main/java/com/parse/Parse.java [hunk 4]
```diff
@@ -608,7 +608,14 @@ private static void initializeParseHttpClientsWithParseNetworkInterceptors() {
     interceptors = null;
   }
 
-  // TODO(mengyan) Make public after we release the interceptor feature
+
+  /**
+   * Add a {@code ParseNetworkInterceptor}. You must invoke
+   * {@code Parse.addParseNetworkInterceptor} before {@code Parse.initialize}. You can add multiple
```
Comment (hallucinogen) [inline Parse/src/main/java/com/parse/Parse.java]: Same comment. It should use `{@link}`

File: Parse/src/main/java/com/parse/Parse.java [hunk 5]
```diff
@@ -608,7 +608,14 @@ private static void initializeParseHttpClientsWithParseNetworkInterceptors() {
     interceptors = null;
   }
 
-  // TODO(mengyan) Make public after we release the interceptor feature
+
+  /**
+   * Add a {@code ParseNetworkInterceptor}. You must invoke
```
Comment (hallucinogen) [inline Parse/src/main/java/com/parse/Parse.java]: Sorry. This one should be `{@link ParseNetworkInterceptor}`

File: Parse/src/main/java/com/parse/Parse.java [hunk 6]
```diff
@@ -608,7 +608,14 @@ private static void initializeParseHttpClientsWithParseNetworkInterceptors() {
...
+   * {@link #addParseNetworkInterceptor(ParseNetworkInterceptor)} before
```
Comment (hallucinogen) [inline Parse/src/main/java/com/parse/Parse.java]: and this one should be `{@code addParseNetworkInterceptor(ParseNetworkInterceptor)}`

File: Parse/src/main/java/com/parse/Parse.java [hunk 7]
```diff
@@ -620,7 +627,13 @@ private static void initializeParseHttpClientsWithParseNetworkInterceptors() {
     interceptors.add(interceptor);
   }
 
-  // TODO(mengyan) Make public after we release the interceptor feature
+  /**
+   * Remove a given {@code ParseNetworkInterceptor}. You must invoke
```
Comment (hallucinogen) [inline Parse/src/main/java/com/parse/Parse.java]: `{@link ParseNetworkInterceptor}`

File: Parse/src/main/java/com/parse/Parse.java [hunk 8]
```diff
@@ -620,7 +627,13 @@ private static void initializeParseHttpClientsWithParseNetworkInterceptors() {
...
+   * {@link #removeParseNetworkInterceptor(ParseNetworkInterceptor)}  before
+   * {@link #initialize(Context)} .
+   * @param interceptor
+   *          The {@code ParseNetworkInterceptor} to be removed.
```
Comment (hallucinogen) [inline Parse/src/main/java/com/parse/Parse.java]: `{@code ParseNetworkInterceptor}`

Comment (wangmengyan95) [inline Parse/src/main/java/com/parse/Parse.java]: er.. based on the rule, this should be `{@link ParseNetworkInterceptor}`, right?

**Comment by hallucinogen** [comment]
LGTM!

**Pull request merged**